src/pages/our-mission.jsx

```diff
@@ -8,127 +9,98 @@ import "../css/Index.css"
-                                    Victoria, Wellington, and now lives in Sydney.
+                                </div>
+                                <div>
+                                    <img src={WillImage} alt="William Parry" />
+                                    <h3>William Parry, CTO</h3>
+                                    <p>
+                                        William brings 15 years of tech experience across enterprise, advertising and
+                                        small businesses including 8 years working with open data in projects and
+                                        hackathons. He has run community coding classes and is passionate about
+                                        empowering disadvantaged people with technology. William went to university at
+                                        Victoria, Wellington, and now lives in Sydney.
                                 </p>
+                                </div>
                             </div>
                         </div>
                     </div>
-                </div>
-                <div className="body-right">
-                    <div className="on-this-page">
-                        <h2>On this page</h2>
-                        <nav>
-                            <ul>
-                                <li>
-                                    <a href="#mission-statement">Mission Statement</a>
-                                </li>
-                                <li>
-                                    <a href="#about-us">About Us</a>
-                                </li>
-                                <li>
-                                    <a href="#our-data">Our data</a>
-                                </li>
-                                <li>
-                                    <a href="#stewardship">Stewardship</a>
-                                </li>
-                                <li>
-                                    <a href="#directors">Directors</a>
-                                </li>
-                            </ul>
-                        </nav>
+                    <div className="body-right">
+                        <div className="on-this-page">
+                            <h2>On this page</h2>
+                            <nav>
+                                <ul>
+                                    {
+                                        missionItems.map(({ title }, idx) => (
+                                            <li key={idx}>
+                                                <a href={`#${title}`}>
+                                                    {title}
+                                                </a>
+                                            </li>
+                                        ))
+                                    }
+                                </ul>
+                            </nav>
```

Comment:
Could be worth breaking the side nav into a react component for now, although I intentionally didn't initially to keep everything simple.

Think this is something we can come back to at a later date when we build out the site a bit more, for now this is good to get it pushed out.

Reply:
I will work on that after adding CMS to the pages. If it's a priority, I can work on this first.

Reply:
Yep no rush I think lets leave it for now and keep things simple.